airbyte-api/src/main/openapi/config.yaml

```diff
@@ -952,7 +952,7 @@ paths:
         content:
           application/json:
             schema:
-              $ref: "#/components/schemas/OperationCreateOrUpdate"
```

Comment:
clarifying naming here. This struct was used for 2 endpoints: `operations/check` and `webackend/connections/update` (which is really an update or create. the check endpoint does not need all the fields the update endpoint needs. So i split them into 2 different structs with only the fields they need and more granular names.